lib/pages/wp-admin/wp-admin-jetpack-page.js

```diff
@@ -13,8 +16,23 @@ export default class WPAdminJetpackPage extends AsyncBaseContainer {
 
 	async connectWordPressCom() {
 		const selector = By.css( 'a.jp-jetpack-connect__button' );
+		const pauseBetweenFocusAttemptsMS = 200;
+		const connectJetpackButtonFocus =
+			"document.getElementsByClassName( 'jp-jetpack-connect__button')[0].focus();";
 		await driverHelper.waitTillPresentAndDisplayed( this.driver, selector );
-		await this.driver.sleep( 1000 );
+		for ( let i = 0; i < explicitWaitMS / pauseBetweenFocusAttemptsMS; i++ ) {
```

Comment:
Is this `for` loop might be replaced with `driver.wait` function? They seem like quite similar